cmake/FDBComponents.cmake

```diff
@@ -59,8 +59,8 @@ endif()
 # Pip
 ################################################################################
 
-find_package(Virtualenv)
-if (Virtualenv_FOUND)
+find_package(Python3)
```

Comment:
We now use `find_package(Python)` for the bindings and ctest and `find_package(Python3)` for the documentation. I think it would be ok to drop Pyhon2 support for these things. In that case this could be consolidated (doesn't necessarily need to be done in this PR).

Reply:
So it would be safe to drop Python2 support throughout? If so, I'll open a separate PR for that.

Reply:
Yes, we are planning to get rid of Python 2 (as support for it ends very soon). But this doesn't need to be done in this PR